### DISCUSSION TITLE
Share link for line of code

### DISCUSSION THREAD

**Discussion by LadislavVasina1** [community/community#39941] [Mobile]
Hi all, loving GitHub app, I was just wondering if there is a possibility to send link to specific line of code from GitHub mobile app (same feature as you can normally do on desktop). If it's is not possible I would suggest it as RFE. Thx a lot.

**Comment by aconsuegra** [comment]
Hi @LadislavVasina1, thanks for using the GitHub Mobile app! Sharing a link to a line of code, and support for deep links to a line (or range)  of code is already supported on Android, but not yet on iOS. Sadly, we don't have an ETA for this feature on iOS.

**Reply by LadislavVasina1** [reply]
Hi thank you very much for blazing fast response. I totally understand. Have a nice day. 😀
